### DISCUSSION TITLE
Set local enviroment

### DISCUSSION THREAD

**Discussion by pedrospeixoto** [pedrospeixoto/MPAS-PXT#8] [General]
With version 7 and below we had to install several local software and therefore I have created a script to load the environmental variables to point correctly to NETCDF, PIO, etc. 

Comments/questions:
- @daniloceano remove the automatic host check at local_software/setup_env.sh
host=`hostname` -->> host='nemo'
I will set this back to automatic hostname, ok?

- In version 8 we no longer need to install the local software from source in /local_software so maybe we should re-write the setup_env.sh completely for version 8?

Any thought @favba @daniloceano ?

**Comment by daniloceano** [comment]
@pedrospeixoto 

` will set this back to automatic hostname, ok?`

Definitely. I changed this option for working locally and forgot to revert back when pushing

`In version 8 we no longer need to install the local software from source in /local_software so maybe we should re-write the setup_env.sh completely for version 8?`

The current version of setup_env.sh worked perfectly for me when I installed version 8, so I'm not sure what could be modified

**Comment by pedrospeixoto** [comment]
Oh, great.

In version 8 a lot of configuration environmental variable are no longer
needed, so it would be just a clean up thing. Let's keep both for now.

Prof. Dr. Pedro da Silva Peixoto
Associate Professor
Applied Mathematics
University of São Paulo (USP <http://www.usp.br/>)

www.ime.usp.br/~pedrosp


Em sex., 27 de out. de 2023 às 16:06, Danilo Couto de Souza <
***@***.***> escreveu:
